libs/vm/src/object.cpp

```diff
@@ -115,143 +115,331 @@ std::size_t Object::GetHashCode()
 
 bool Object::IsEqual(Ptr<Object> const & /* lhso */, Ptr<Object> const & /* rhso */)
 {
-  RuntimeError(std::string(__func__) + "operator not implemented");
+  RuntimeError(std::string(__func__) + ": operator not implemented");
   return false;
 }
 
 bool Object::IsNotEqual(Ptr<Object> const & /* lhso */, Ptr<Object> const & /* rhso */)
 {
-  RuntimeError(std::string(__func__) + "operator not implemented");
+  RuntimeError(std::string(__func__) + ": operator not implemented");
```

Comment:
" :   .... ", a space right after " and before : would be neater.
And to simplify such corrections let the phrase be a 

static constexpr char * OP_NOT_IMPL = " .... " ;

used across all functions.